index.html

```diff
@@ -0,0 +1,656 @@
+    <!DOCTYPE html>
+    <html lang="en">
+
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      <title>New York Times</title>
+      <script src="https://kit.fontawesome.com/fc8cc3c5c9.js"></script>
+      <link rel="stylesheet" href="css/style.css">
+    </head>
+
+    <body>
+      <!-- Main pag header -->
+      <header>
+        <nav id="header-nav">
```

Comment:
Prefer HTML classes over ID. This helps to keep code DRY as it promotes reusability of code.